### PR TITLE
Fix up import statements and also.

### PR DESCRIPTION
**Pull request by tonyseek**

### CODE REVIEW

**Comment by emmett9001** [comment]
Hi, thanks for the input! I'm going over this pull request now. It would be helpful if you could provide a description of your goals with these changes. What is the main purpose of this pull request?

File: requirements.txt [hunk 1]
```diff
@@ -1,6 +0,0 @@
-rdflib
```
Comment (emmett9001) [inline requirements.txt]: Why is this file deleted? It's useful for local testing. `setup.py develop` can also be used for this - is that what you had in mind?

File: schemato/schemadef.py [hunk 2]
```diff
@@ -36,10 +39,11 @@ def _read_schema(self):
         """return the local filename of the definition file for this schema
         if not present or older than expiry, pull the latest version from
         the web at self._ontology_file"""
-        cache_filename = os.path.join(settings.CACHE_ROOT, "%s.smt" % self._representation)
-        log.info("Attempting to read local schema at {}".format(cache_filename))
+        cache_filename = os.path.join(
+            CACHE_ROOT, "%s.smt" % self._representation)
+        log.info("Attempting to read local schema at %s" % cache_filename)
```
Comment (emmett9001) [inline schemato/schemadef.py]: Why are you using the old `%`-based string formatting here? `str.format()` is generally my preference since it's more expressive.

File: schemato/schemadef.py [hunk 3]
```diff
@@ -61,10 +65,11 @@ def parse_ontology(self):
         """place the ontology graph into a set of custom data structures
         for use by the validator"""
         start = time.clock()
-        log.info("Parsing ontology file for {}".format(self.__class__.__name__))
+        log.info("Parsing ontology file for %s" % self.__class__.__name__)
```
Comment (emmett9001) [inline schemato/schemadef.py]: Same comment as above, I prefer `str.format`

Comment (dan-blanchard) [inline schemato/schemadef.py]: For logging, you really should just use a `,` instead of `%` and use old-style formatting.  That makes it so the string doesn't actually get formatted unless that level of logging happens.

Comment (emmett9001) [inline schemato/schemadef.py]: Oh thanks, I didn't know that.

Comment (dan-blanchard) [inline schemato/schemadef.py]: With Python 3 there's a way to do the same thing for new-style formatting, but it requires setting the optional `style` parameter to `{`.  Only works in 3.2+ though, so I almost always end up using old-style formatting for logging.

File: schemato/schemato.py [hunk 4]
```diff
@@ -43,31 +47,23 @@ def validate(self):
...
     def _load_validators(self):
-        def import_module(name):
-            m = __import__(name)
-            for n in name.split(".")[1:]:
-                m = getattr(m, n)
-            return m
-        # include the parent directory in the path, to allow relative imports of
-        # modules from settings
-        os.sys.path.append(os.path.dirname(os.path.abspath(__file__)))
-        self.validators = set()
```
Comment (emmett9001) [inline schemato/schemato.py]: This line was deleted and needs to be re-added:

https://gist.github.com/emmett9001/a3740c1e195abb5b3dd4

File: schemato/validator.py [hunk 5]
```diff
@@ -56,8 +58,10 @@ def _should_ignore_predicate(self, predicate):
         ignored_predicates = ['type', 'item', 'first', 'rest']
         return self._field_name_from_uri(predicate) in ignored_predicates
 
-    def _check_triple(self, (subj, pred, obj)):
+    def _check_triple(self, args):
```
Comment (emmett9001) [inline schemato/validator.py]: This could use a more descriptive name, like `triple`

File: schemato/validator.py [hunk 6]
```diff
@@ -178,20 +188,23 @@ def _superclasses_for_subject(self, graph, typeof):
                 break
         return classes
 
-    def _is_instance(self, (subj, pred, obj)):
+    def _is_instance(self, args):
```
Comment (emmett9001) [inline schemato/validator.py]: `args` -> `triple`

**Comment by emmett9001** [comment]
This is great! Looks like most of it is style fixes, which I appreciate. The new way of doing plugin imports is much better than the old, thanks for that as well. I made a few comments, but they're all pretty minor so I may end up fixing them myself post-merge.

One problem I notice is that the tests now fail due to relative imports.
